src/transaction/payment.ts

```diff
@@ -85,29 +85,57 @@ function createMaximalAmount(amount: Amount): Amount {
   return _.assign({}, amount, {value: maxValue})
 }
 
+function normalizeAddress(address: string, tag: number | undefined, verification?: string):
+    {classicAddress: string, tag?: number} {
+  let addressClassic
```

Comment:
small nit: you call everything else `classicAddress` and this is `addressClassic`. I'd value consistency unless you think that there's something I'm missing.

Reply:
This one is meant to be more like `address (classic)` or a reference to its origin (the function's `address` parameter). I'm fine with naming it classicAddress though.